bib/internal/util/util.go

```diff
@@ -26,3 +26,48 @@ func RunCmdSync(cmdName string, args ...string) error {
 	}
 	return nil
 }
+
+func checkContainerImageExists(storageRef string) (bool, error) {
+	cmd := exec.Command("skopeo", "inspect", storageRef)
+	err := cmd.Run()
+	if err != nil {
+		if cmd.ProcessState.ExitCode() == 1 {
+			return false, nil
+		}
+		return false, err
+	}
+	return true, err
+}
+
+func pullContainerImage(imgref, storageRef string) error {
+	return RunCmdSync(
+		"skopeo",
+		"copy",
+		fmt.Sprintf("docker://%s", imgref),
+		storageRef,
+	)
+}
+
+func EnsureContainerIsInStore(imgref string) error {
+	driver := "overlay"
+	graphRoot := "/var/lib/containers/storage"
+	runRoot := "/run/containers/storage"
+	storageImgRef := fmt.Sprintf("containers-storage:[%s@%s+%s]%s", driver, graphRoot, runRoot, imgref)
+
+	exists, err := checkContainerImageExists(storageImgRef)
+	if err != nil {
+		return fmt.Errorf("Failed to check if container is in local store: %v", err)
```

Comment:
Here and elsewhere

- Use `%w` and not `%v` for wrapping errors
- Go style does not capitalize the first word (not my favorite, but it is what it is)